otj-kafka-mirrormaker/src/main/java/com/opentable/kafka/mirrormaker/RenameTopicHandler.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.opentable.kafka.mirrormaker;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.producer.ProducerRecord;
+
+import kafka.consumer.BaseConsumerRecord;
+import kafka.tools.MirrorMaker;
+// https://github.com/slavirok/mirrormaker-rename-topic-handler, https://github.com/opencore/mirrormaker_topic_rename mildly changed
+// to run
+//kafka-mirror-maker --consumer.config consumer.cfg --producer.config producer.cfg --whitelist "source1|source2" \
```

Comment:
Lightly changed version of these links. I considered more elaborate scheme but passing the arg is limited to one string with a single arg constructor. I suppose we could go json or yaml and include those deps too, at thhe cost of deployment complication, but I'm not sure I'd agree it's worth it.